frontend/src/components/charts/MainChartPage.js

```diff
@@ -95,7 +121,22 @@ function MainChartPage() {
       <div className="mainCard">
         <div className="titleCard">
           <Title text="Przewidywany stan konta" />
+          <button className="openModalButton" onClick={openModal}>
+            Zaplanuj
```

Comment:
Na moim laptopie przycisk jest lekko za daleko przesuniety w prawo. Nie wiem czy to problem rozdzielczości, czy cos?
![image](https://user-images.githubusercontent.com/69384237/167034650-c0bd7903-4904-4de1-8b80-4c245d48d23b.png)

Reply:
Tak, chodzi o rozdzielczość. U mnie wygląda tak:
![zaplanuj1](https://user-images.githubusercontent.com/48785655/167075366-0811f12f-673b-4f0f-bfba-49b279513d7e.png)
![zaplanuj2](https://user-images.githubusercontent.com/48785655/167075374-5eabfa93-74c3-495a-8504-e0b7452b466e.png)

Żeby to naprawić należałoby zdefiniować punkty krytyczne przy których button zmienia swoje położenie w odpowiedzi na skalowanie ekranu (tzw. responsywność).
Myślę, że jak na razie możemy ustalić, kto prezentuje klientowi wystarczy zrobić tak, żeby u niego to ładnie wyglądało xd

Reply:
Okay